Docs: Getting Started Guide for Newbies

janklostermann opened the pull request:
This is a new `Getting Started for Newbies` guide, as inspired by the conversation at https://community.openems.io/t/aller-anfang-ist-schwer/5667.
It is supposed to help someone completely new to OpenEMS to get a first user experience, even if not daring to start developing - to get a first familiarity with the system.

As I am still new to OpenEMS myself, I might be able to take care of pain-points of newbies. But it also means, that I made some assumptions here and there, that might not be fully correct. Please be aware and fix.

I have integrated @DerWahreKlinki 's script for a setup without docker. It might need a better place for it.

One strange thing I had to do: Make the factor in the datasource2 for the non-regulated consumption negative (-10000) to achieve positiv consumption in the example. I hope that makes sense. If not, please fix.

Beyond that: I wanted to explain, what the colored ring segments of the energy meter mean, but I could not find out myself...

TL;DR: There is still room for improvement. Thanks for considering anyhow.

janklostermann committed:
first draft

janklostermann committed:
deployment script for raspberry.pi from klinki (see https://community.openems.io/t/edge-ui-deployment-ein-grober-leitfaden-raspberry-pi/729/44)

janklostermann committed:
temporary step

janklostermann committed:
after first review

janklostermann committed:
fixing the header

janklostermann committed:
after testing it through

da-Kai commented:
Hi @janklostermann, thanks for the contribution. Did you already check PR #3268? Do your changes relate or overlap with it?

da-Kai reviewed (changes requested):
Hi @janklostermann,
#3268 is already merged. While that PR mainly extrended the developmer guide, it also simplified the getting started: [Getting Started :: Open Energy Management System](https://openems.github.io/openems.io/openems/latest/gettingstarted.html). 
The only difference i see, is that your newbie doc uses Docker instead of the .jar files. please correct me if I am wrong.

Also your documentation are currently not referenced. You also need to adjust `doc/modules/ROOT/nav.adoc`.

janklostermann commented on doc/modules/ROOT/pages/deployment-raspi.sh:
I fully agree that the Raspi-deployment fits better in .../edge/deploy/. 
Do I interpret you correctly that you suggest to create a `doc/modules/ROOT/pages/edge/deploy/raspberrypi.adoc` with explanations (might be just taken from the [forum](https://community.openems.io/t/edge-ui-deployment-ein-grober-leitfaden-raspberry-pi/729/40) and the related PDF) and the script referenced in the .adoc placed at `tools/deploy/rpi/deployment-raspi.sh` ?

da-Kai commented:
The way your current doc is written i would argue that if someone is tech-savvy enough, to setup linux and use the bash shell, they should be able to work with artifacts like .jar or .zip. 

If one manged to setup, by only following docs and AI-generated instructions, they are likely to hit a wall sooner rather than later.

The only genuinely easy alternatives I see would be setting up via Docker Desktop or actually providing portable .exe and .AppImage files.

janklostermann committed:
Extend Energy Monitor explanation

Clarified the interpretation of gauge bars in the UI including the comment of @Sn0w3y in https://github.com/OpenEMS/openems/issues/3387#issuecomment-3478153201

janklostermann committed:
Merge branch 'develop' into docs/newbieGettingStarted
